app/helpers/budgets_helper.rb

```diff
@@ -1,5 +1,23 @@
 module BudgetsHelper
 
+  def csv_params
+    csv_params = params.clone.merge(format: :csv)
+    csv_params.delete(:page)
+    csv_params
+  end
+
+  def investment_selected_link_options(investment)
```

Comment:
Great refactoring 👍 
Keep in mind that if you run into a refactoring to do, it's nicer to do it in a separate PR